cms/earn/strategies.json

```diff
@@ -2639,7 +2719,119 @@
+      "unlisted": false,
+      "disabled": false,
+      "message": "Buys Liquid Staked Assets at a discount and performs the unbonding to earn the delta.",
+      "depositDenoms": [
+        {
+          "coinMinimalDenom": "uosmo",
+          "_comment": "Osmosis"
+        }
+      ],
+      "positionDenoms": [
+        {
+          "coinMinimalDenom": "factory/osmo15fqmdl8lfl9h0qflljd63ufw9j2m7xmsk3hu5vsn8xpta4hk5chqt7mddc/redemption-vault-uosmo",
+          "_comment": "OSMO Locust Redemption Vault Token"
+        }
+      ],
+      "rewardDenoms": [],
+      "categories": [
+        "Correlated",
+        "Blue Chip"
+      ]
+    },
+    {
+      "id": "osmo1reyz7pwu7y9e7lmzqg6j4h7jcv32du7n7jhnk2lz93a9lxr56ess2qtgzl",
+      "name": "TIA Redemption Rate Arbitrage",
+      "platform": "Margined",
+      "type": "Liquid Staking",
+      "method": "margined-locust-redemption",
+      "link": "https://trade.margined.io/vaults/tia-redemption-rate",
+      "tvl": "https://vaults.margined.io/api/v1/${id}/tvl",
+      "apr": "https://vaults.margined.io/api/v1/${id}/apr",
+      "balance": "https://vaults.margined.io/api/v1/${id}/balance/${address}",
+      "geoblock": "https://vaults.margined.io/api/v1/${id}/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=1058195732#gid=1058195732",
+      "startDateTimeUtc": "2024-07-31T00:00:00Z",
+      "unlisted": false,
+      "disabled": false,
+      "message": "Buys Liquid Staked Assets at a discount and performs the unbonding to earn the delta.",
+      "depositDenoms": [
+        {
+          "coinMinimalDenom": "ibc/D79E7D83AB399BFFF93433E54FAA480C191248FC556924A2A8351AE2638B3877",
+          "_comment": "Celestia"
+        }
+      ],
+      "positionDenoms": [
+        {
+          "coinMinimalDenom": "factory/osmo1reyz7pwu7y9e7lmzqg6j4h7jcv32du7n7jhnk2lz93a9lxr56ess2qtgzl/redemption-vault",
+          "_comment": "TIA Locust Redemption Vault Token"
+        }
+      ],
+      "rewardDenoms": [],
+      "categories": [
+        "Correlated",
+        "Blue Chip"
+      ]
+    },
+    {
+      "id": "osmo1hvl5kj4xzdj4udxjv2dzk2zfqhzkd9afqygwq3t84tn53e0250zqrltj48",
+      "name": "ATOM Redemption Rate Arbitrage",
+      "platform": "Margined",
+      "type": "Liquid Staking",
+      "method": "margined-locust-redemption",
+      "link": "https://trade.margined.io/vaults/atom-redemption-rate",
+      "tvl": "https://vaults.margined.io/api/v1/${id}/tvl",
+      "apr": "https://vaults.margined.io/api/v1/${id}/apr",
+      "balance": "https://vaults.margined.io/api/v1/${id}/balance/${address}",
+      "geoblock": "https://vaults.margined.io/api/v1/${id}/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=634542877#gid=634542877",
```

Comment:
Please replace with this URL: https://docs.google.com/spreadsheets/d/1RlH1KT6EUJCY6ngJ_8TdPp9csF9T9iB8euyNlGSM17Y/edit?usp=sharing